go/host/p2p/p2p.go

```diff
@@ -247,7 +249,7 @@ func (p *p2pImpl) broadcast(msg message) error {
 	var wg sync.WaitGroup
 	for _, address := range p.peerAddresses {
 		wg.Add(1)
-		go p.sendBytes(&wg, address, msgEncoded)
+		go p.sendBytesWithRetry(&wg, address, msgEncoded) //nolint: errcheck
```

Comment:
Ok to fail silently here? I guess we're supposed to track communications metrics with that gauge thing so maybe it gets caught by that?

Reply:
it's not ok, but there is some logging going on.
And it will get addressed hopefully by the p2p work